assets/css/custom.css

```diff
@@ -316,6 +316,12 @@ a.link.white {
 .nav-section a.register-now {
 	color: #EB6851;
 }
+
+.nav-section button {
+	color: #EB6851;
+	border: none;
+	background-color: #172751;
+}
```

Comment:
instead of targeting `<button>` I would suggest targeting `<tito-button>` and `.tito-tickets-button` since this is how it renders on the page:

``` html
<tito-button event="write-speak-code/2017-conference"><button class="tito-tickets-button" id="button-tito-write-speak-code-2017-conference-1">Register Now</button></tito-button>
```